modules/installation-localzone-generate-k8s-manifest.adoc

```diff
@@ -56,22 +56,6 @@ spec:
 EOF
 ----
 
-.. If you are using the `OpenShift SDN` network plugin, enter the following command:
```

Comment:
This block needs to be simplified, as `If you are using the `OVN-Kubernetes` network plugin,` appears above, and now that is always true.